frontend_lib/README.md

```diff
@@ -21,59 +66,128 @@ This will produce two UMD modules:
 - `dist/tracim_frontend_lib.lib.js`: the library part
 - `dist/tracim_frontend_lib.style.js`: the styling part
 
-The Webpack configuration used to build the standalone bundle is `webpack.config.js`.
+The Webpack configuration used is [webpack.config.js](./webpack.config.js).
 
-## When to use this bundle
+### When to use this bundle
 
-This bundle should be used when building a Tracim app outside of Tracim.
+This bundle should be used for using Tracim lib's features outside of Tracim.
 
-# Building without shared vendors, using externals
+#### Pros
+- it is a conventional build
+- it is exported as a UMD module
 
-This build can be included using a `<script>` tag.
-It does not come with all its dependencies.
-Some of them are in the `tracim_frontend_vendors` bundle.
-The point of this way of building is to share the `tracim_fontend_lib` and common `dependencies` across the frontend, the apps and the lib in Tracim.
+#### Cons
+- it cannot use the tracim_vendors feature: so it will bundle dependencies that are already available in Tracim
 
-The build can be used from a package (a Tracim frontend app) by accessing the global variable `tracim_frontend_lib` or in a package built with Webpack as long as:
 
-- `tracim_frontend_lib` is listed in the `externals` property of the Webpack configuration
-- all the libraries in `tracim_frontend_vendors` are also listed in `externals`
-- `tracim_frontend_vendors` and `tracim_frontend_libs` are included using the `<script>` tag before the app is loaded. In this configuration, `tracim_frontend_lib` expects the global object `tracim_frontend_vendors` to be available.
+## Frontend unit tests
 
-This can be done by using the `externals.json` file built when bundling `tracim_frontend_vendors`.
+Frontend unit tests uses the standalone build, hence the need to build it when running [build_full_frontend.js](../build_full_frontend.js).  
+Using the optimized build which does not include the shared vendors does not work: `tracim_frontend_lib` needs to be compatible with CommonJS, and needs to access its dependencies. It is, however, impossible to make a `tracim_frontend_vendors` global object available for `tracim_frontend_lib` when using CommonJS modules, since their is no global namespace shared between the modules (in Node.js at least).
```

Comment:
```suggestion
Using the optimized build which does not include the shared vendors does not work: `tracim_frontend_lib` needs to be compatible with CommonJS, and needs to access its dependencies. It is, however, impossible to make a `tracim_frontend_vendors` global object available for `tracim_frontend_lib` when using CommonJS modules, since there is no global namespace shared between the modules (in Node.js at least).
```